sdk/communication/azure-communication-chat/azure/communication/chat/_shared/models.py

```diff
@@ -216,24 +279,54 @@ def identifier_from_raw_id(raw_id: str) -> CommunicationIdentifier:
         return MicrosoftTeamsUserIdentifier(
             user_id=suffix,
             is_anonymous=False,
-            cloud='PUBLIC'
+            cloud=CommunicationCloudEnvironment.PUBLIC
         )
     elif prefix == '8:dod:':
         return MicrosoftTeamsUserIdentifier(
             user_id=suffix,
             is_anonymous=False,
-            cloud='DOD'
+            cloud=CommunicationCloudEnvironment.DOD
         )
     elif prefix == '8:gcch:':
         return MicrosoftTeamsUserIdentifier(
             user_id=suffix,
             is_anonymous=False,
-            cloud='GCCH'
+            cloud=CommunicationCloudEnvironment.GCCH
         )
     elif prefix in ['8:acs:', '8:spool:', '8:dod-acs:', '8:gcch-acs:']:
         return CommunicationUserIdentifier(
             id=raw_id
         )
+    elif prefix == '28:gcch-global:':
```

Comment:
I guess it's time to move out hardcoded strings into constants and reuse them everywhere needed :)

Reply:
Done